composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": ">=7.4 | ^8.0",
         "ext-json": "*",
-        "guzzlehttp/guzzle": "~6.0",
+        "guzzlehttp/guzzle": "~7.0",
```

Comment:
Can you allow both 6 and 7 please?

Reply:
Of course, completely overlooked the bc break.